skiko/buildSrc/src/main/kotlin/properties.kt

```diff
@@ -162,7 +156,7 @@ class SkikoProperties(private val myProject: Project) {
                 System.getenv()["SKIA_DIR"]
                 ?: System.getProperty("skia.dir")
                 ?: myProject.findProperty("skia.dir")?.toString()
-            )?.let { File(it) }?.takeIf { it.isDirectory }
+            )?.let { File(it) }
```

Comment:
Was this check for `isDirectory` removed intentionnaly?

Reply:
Without validation  there was always a possibility to just have a seemingly valid local build which in actuality was remote build. I  totally agree it's better to keep it in getter. Pushed